### ISSUE TITLE
CI-buildwheel is failing on Windows 64 bits

### ISSUE THREAD

**Issue by kif** [silx-kit/pyFAI#2162]
Hint from VFN:
https://github.com/diffpy/pyobjcryst/issues/33

**Comment by EdgarGF93** [comment]
I tested the wheels for windows cp37 and cp38, and got the error in both cases:

```
(py37) C:\Users\edgar1993a>python -c "from pyFAI.test.test_histogram import TestHistogram1d"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\edgar1993a\py37\lib\site-packages\pyFAI\test\test_histogram.py", line 44, in <module>
    from ..ext.histogram import histogram, histogram2d, histogram2d_engine
ImportError: DLL load failed: The specified module could not be found.
```

By running this test script:
```import sys
import dlltracer
import ctypes


with dlltracer.Trace(out=sys.stdout):
    ctypes.CDLL("C:\\Users\\edgar1993a\\py38\\lib\\site-packages\\pyFAI\\ext\\histogram.cp38-win_amd64.pyd")

```
I get:
```
(py38) C:\Users\edgar1993a>python T:\edgar\test_dll.py
LoadLibrary \Device\HarddiskVolume7\Windows\System32\kernel.appcore.dll
LoadLibrary \Device\HarddiskVolume7\Users\edgar1993a\py38\Lib\site-packages\pyFAI\ext\histogram.cp38-win_amd64.pyd
Failed \Device\HarddiskVolume7\Users\edgar1993a\py38\Lib\site-packages\pyFAI\ext\histogram.cp38-win_amd64.pyd
Traceback (most recent call last):
  File "T:\edgar\test_dll.py", line 7, in <module>
    ctypes.CDLL("C:\\Users\\edgar1993a\\py38\\lib\\site-packages\\pyFAI\\ext\\histogram.cp38-win_amd64.pyd")
  File "C:\python\3.8\lib\ctypes\__init__.py", line 373, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'C:\Users\edgar1993a\py38\lib\site-packages\pyFAI\ext\histogram.cp38-win_amd64.pyd' (or one of its dependencies). Try using the full path with constructor syntax.
```

**Comment by EdgarGF93** [comment]
similar to this issue:
https://stackoverflow.com/questions/59330863/cant-import-dll-module-in-python/64472088#64472088

**Comment by EdgarGF93** [comment]
One difference between the (bad) wheel generated by github and the wheel generated using cibuildwheel on the local windows is that the binary files in the directory: .../pyFAI/ext are **.dll.a** in the case of the bad wheel and **.lib** files in the good wheel

**Comment by kif** [comment]
I found the `histogram.pxd` extension was trying to load the `libgomp.dll` (not existing on the target computer). 
We decided in the silx project (many years back) to distribute only wheels with OpenMP disabled on windows and macos to avoid this kind of issues. If users need best performances, they can (and should be able to) rebuild their packages themselves.

This indicates the wheel was built with the wrong compiler or the wrong library linked.
`cibuildwheel` is pretty opaque on what it is doing and we have little control on the operating system provided by github-actions. Maybe the simplest would be to setup an environment variable disabling the OpenMP extension. Let's check how to do that with `meson` ...
